src/css/controls/imports/slider.less

```diff
@@ -213,3 +213,11 @@
         height: 11px;
     }
 }
+
+.jw-horizontal-volume-container {
+    display: none;
+
+    .jw-flag-audio-player & {
+        display: flex;
+    }
+}
```

Comment:
Voiceover was actually only detecting the `horizontal-volume-slider` and announcing that as opposed to the `vertical-volume-slider`.

Reply:
How is `jw-horizontal-volume-container` currently hidden - Is it just with `width: 0`?

Can you link to an audio-only test here/ in JIRA (maybe add the right label) so TE is sure to cover it in tests?

Reply:
> How is `jw-horizontal-volume-container` currently hidden - Is it just with `width: 0`?

Yeah. `width: 0` and `tabindex = -1`. Looks like the only time it "opens" is with audio players.
https://github.com/jwplayer/jwplayer/blob/4763f810c473157043d032069611401762f782ab/src/css/controls/flags/audioplayer.less#L78-L85

> Can you link to an audio-only test here/ in JIRA (maybe add the right label) so TE is sure to cover it in tests?

Updated JIRA ticket and added `core` label as well. Not sure if there is another label to add.

Reply:
`@audio_only` is the label that shows up in `.feature` files if you search for audio only test cases. We need to add the label to JIRA. cc @aizaztoppa